lib/gherkin_lint/linter/tag_constraint.rb

```diff
@@ -0,0 +1,32 @@
+module GherkinLint
+  # Mixin to lint for tags that have certain string contraints
+  module TagConstraint
```

Comment:
you require that every Scenario/Feature has a Ticket-Reference. I do not require that for my Projects, but I would require, that just a certain pattern for Tags is used. Like @skip or @MCC-[\d]+. Could we express this as well?